test/docfx.Test/data/validation/taxonomy-allowlists.json

```diff
@@ -0,0 +1,4540 @@
+{
```

Comment:
This file is big, can we shrink it a little bit?

Reply:
OK, it's a test file, I will shrink it.